reference/configuration.html.markerb

```diff
@@ -616,6 +616,73 @@ This value is only used by `fly launch` and `fly deploy` commands in case they n
 
 The value can be a string with units, like `initial_size = "30GB"` for example. The value can also be an integer, like `initial_size = 30` for example, with GB as the default unit.
 
+## The `vm` section
+
+This section defines the compute requirements required by the virtual machines used for the application.
+
+The default virtual machine size is `shared-cpu-1x` but it is not enforced if not specified in fly.toml. It means commands like `fly deploy` won't attempt to update the compute requirements (or size) for a machine. `fly scale` commands will use existing machine to infer new machine sizes and in the event that there are no machines, they will fallback to `shared-cpu-1x`.
+
+If you care about a predictable _scaling up from zero_ case, or if you have different compute requirements for different process groups, then it is highly recommended to add this section to fly.toml.
```

Comment:
I think we need to explicitly repeat here that `fly scale count` and `fly deploy` will use these settings.

Reply:
check this please https://github.com/superfly/docs/pull/1179/commits/a64821beefdf8cad71ab5896c40d67bc7c1763e9